pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -2171,3 +2194,17 @@ func hotUnplugHostDevices(virConn cli.Connection, dom cli.VirDomain) error {
 	}
 	return nil
 }
+
+// check whether VMI has a certain condition
+func vmiHasCondition(vmi *v1.VirtualMachineInstance, cond v1.VirtualMachineInstanceConditionType) bool {
```

Comment:
i think a function already exists for this

```HasCondition()``` in ```/pkg/controller/conditions.go``` 

can that be reused?

Reply:
yes. In general, I didn't want to import this into virt-launcher, that's why I reimplemented it here.